src/client/components/Delete_Comfim/ModalList.component.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Button from '../button/button.component';
+
+const ModalList = ({ isShowing, hide, onClick }) => {
+  if (isShowing) {
+    return ReactDOM.createPortal(
+      <div>
+        <div className="modal-overlay" />
+        <div
+          className="modal-wrapper"
+          aria-modal
+          aria-hidden
+          tabIndex={-1}
+          role="dialog"
+        >
+          <div className="modal-header">
+            <span
+              type="button"
+              className="modal-close-button"
+              aria-label="Close"
+              onClick={hide}
+              aria-hidden="true"
+            >
+              &times;
+            </span>
+          </div>
+          <div className="modal">
+            <h4 className="text-headline">DELETE BATCH</h4>
+            <p className="p1">Are you sure?</p>
+            <p className="p2">Do you really want to delete the batch?</p>
+            <div className="buttons-wrapper">
+              <Button type="danger" size="large" onClick={onClick}>
```

Comment:
About this `onClick` function: we don't have the endpoints ready yet, but eventually when this button is clicked, it will delete the crop from the DB. `ModalList` component should receive that function through props, and you can call it `onDelete` or something similar. You can pass it as, for example, `onDelete={() => null}` since we don't yet know exactly what it will look like.

Reply:
yes